Author: Eugenio "g7" Paolantonio <me@medesimo.eu>

This simple patch ensures that the mpris2 plugin is enabled by default.
Index: pragha/src/pragha-plugins-engine.c
===================================================================
--- pragha.orig/src/pragha-plugins-engine.c
+++ pragha/src/pragha-plugins-engine.c
@@ -80,7 +80,7 @@ pragha_plugins_engine_startup (PraghaPlu
 {
 	PraghaPreferences *preferences;
 	gchar **loaded_plugins = NULL;
-	const gchar *default_plugins[] = {"notify", "song-info", NULL};
+	const gchar *default_plugins[] = {"notify", "mpris2", "song-info", NULL};
 
 	CDEBUG(DBG_PLUGIN,"Plugins engine startup");
 
